airbyte-webapp/src/utils/links.ts

```diff
@@ -29,8 +29,10 @@ export const links = {
   webhookVideoGuideLink: "https://www.youtube.com/watch?v=NjYm8F-KiFc",
   webhookGuideLink: `${BASE_DOCS_LINK}/operator-guides/configuring-sync-notifications/`,
   cronReferenceLink: "http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html",
-  cloudAllowlistIPsLink: `${BASE_DOCS_LINK}/cloud/getting-started-with-airbyte-cloud/#allowlist-ip-address`,
+  cloudAllowlistIPsLink: `${BASE_DOCS_LINK}/cloud/getting-started-with-airbyte-cloud/#allowlist-ip-addresses`,
```

Comment:
I noticed the URL for the docs was changed as well - not 404-ing, but the link to the heading was broken 🙂